script/base.sh

```diff
@@ -14,7 +14,8 @@ sudo apt-get install -y \
     apache2-utils \
     net-tools \
     python3-dev \
-    python3-pip
+    python3-pip \
```

Comment:
please add these two commands in [vagrant setup](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/deploy/vagrant/make_boxes/dtaas/developer.sh) as well. It would be great to confirm the working of [vagrant setup](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/deploy/vagrant/make_boxes/dtaas/README.md) after making these changes.

Reply:
The developer.sh file doesn't install python. So did you mean deploy/vagrant/make_boxes/dtaas/user.sh in the same directory?

Reply:
@prasadtalasila ?